### DISCUSSION TITLE
Using authorizer in NodeJS Express app

### DISCUSSION THREAD

**Discussion by sr258** [authorizerdev/authorizer#311] [Q&A]
I'm currently checking out authorizer and from it's feature set it looks like it has all I need. However, the documentation is a bit thin on how to actually integrate authorization into an app. I use NodeJS and Express. How would I use authorizer with this technology? I imagine I could write a Passport strategy, but I'm a bit lost on how the authentication flow works with authorizer.

**Comment by lakhansamani** [comment]
@sr258 sorry for the delay in reply.
I will share an example of how to use authorizer in NODEJS as auth middleware to make authorized API calls

**Comment by lakhansamani** [comment]
@sr258 check the example here: https://github.com/lakhansamani/authorizer-express-react
